spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/rsocket/RSocketMessageHandlerCustomizer.java

```diff
@@ -0,0 +1,18 @@
+package org.springframework.boot.autoconfigure.rsocket;
+
+import org.apache.catalina.connector.Connector;
+import org.springframework.boot.web.embedded.tomcat.ConfigurableTomcatWebServerFactory;
+import org.springframework.messaging.rsocket.RSocketStrategies;
+import org.springframework.messaging.rsocket.annotation.support.RSocketMessageHandler;
+import org.springframework.util.RouteMatcher;
+
+/**
+ * @author Aarti Gupta
+ * Callback interface that can be used to customize a RSocketMessageHandler {@link Connector}.
+ */
+@FunctionalInterface
+public interface RSocketMessageHandlerCustomizer {
+
+	RSocketMessageHandler setRouteMatcher(RouteMatcher handler);
```

Comment:
To be consistent with other customizers, the method shouldn't return anything but be `void` instead.